### PR TITLE
Add "Do you know your NHS number?" question

### PR DESCRIPTION
**Pull request by issyl0**
No spacing:

Screenshot 2020-03-21 at 15 52 41

With spacing (like prototype):

Screenshot 2020-03-21 at 16 06 11

Links to dummy pages before/after.

### CODE REVIEW

**Review by leenagupte — DISMISSED** [review]
Not a blocker, but there is supposed to be an empty line after the first line of the hint text.

**Comment by issyl0** [comment]
Frontend devs (@injms @alex-ju) please tell me better ways of spacing paragraphs. If I made them actual paragraphs there was too much space between the title and the first line of the hint, so I hacked it with `<br />`. 🤢

**Comment by issyl0** [comment]
Also this needs an `elsif` to account for skipping the (as yet unwritten) NHS number question if the user doesn't know their NHS number... I'll borrow from the business form...

File: config/locales/en.yml [hunk 1]
```diff
@@ -73,6 +73,17 @@ en:
           label: "Yes, I’ve had problems with drug addiction or alcohol addiction (alcoholism)"
         option_no:
           label: "No"
+    know_nhs_number:
+      title: "Do you know your NHS number?"
+      hint: |
+        An NHS number is a 10 digit number, like 485 777 3456.
+        <br /><br />
```
Comment (alex-ju) [inline config/locales/en.yml:115]: This is rather atypical as we don't tend to have hints with multiple paragraphs. I think this should do for now.

Comment (issyl0) [reply config/locales/en.yml:115]: Thanks Alex!

**Comment by issyl0** [comment]
> Also this needs an `elsif` to account for skipping the (as yet unwritten) NHS number question if the user doesn't know their NHS number... I'll borrow from the business form...

On this, [the business form has](https://github.com/alphagov/govuk-coronavirus-business-volunteer-form/blob/f53e993aaca4cf0a58d233d72bf4b0869c5d5c04/app/controllers/coronavirus_form/offer_transport_controller.rb#L25-L26):

```ruby
    elsif session[:offer_transport] == I18n.t("coronavirus_form.#{PAGE}.options.option_yes.label")
      redirect_to controller: "coronavirus_form/transport_type", action: "show"
```

But I can't figure out what the `yes` or `no` options are for this one. 'Cause we want to go to the next question after the "enter your NHS number" one if the user says no.

File: app/controllers/coronavirus_form/know_nhs_number_controller.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    end
+  end
+
+private
+
+  PAGE = "know_nhs_number"
+  NEXT_PAGE = "check_answers"
+
+  def previous_path
+    "/" # TODO
```
Comment (huwd) [inline app/controllers/coronavirus_form/know_nhs_number_controller.rb]:
```suggestion
   coronavirus_form_contact_details_path
```

I think this one has been merged now

File: config/locales/en.yml [hunk 3]
```diff
@@ -108,6 +108,17 @@ en:
         label: "Phone number (for text messages)"
       email:
         label: "Email address"
+    know_nhs_number:
+      title: "Do you know your NHS number?"
+      hint: |
+        An NHS number is a 10 digit number, like 485 777 3456.
```
Comment (huwd) [inline config/locales/en.yml:114]: is spacing too big if we use `<p>` tags here?

**Review by huwd — APPROVED** [review]
One question and a suggestion, but looks great!

File: config/locales/en.yml (hunk 3 above)
Comment (issyl0) [reply config/locales/en.yml:114]: Yes.

File: app/controllers/coronavirus_form/know_nhs_number_controller.rb (hunk 2 above)
Comment (issyl0) [reply app/controllers/coronavirus_form/know_nhs_number_controller.rb]: Thanks - applied.

**Pull request merged**